components/ingest-service/backend/elastic/migration.go

```diff
@@ -33,7 +34,7 @@ func (es *Backend) DoesAliasExists(ctx context.Context, aliasName string) bool {
 
 // DoesIndexExists - does the index 'indexName' exists in elasticsearch
 func (es *Backend) DoesIndexExists(ctx context.Context, indexName string) (bool, error) {
-	return es.client.IndexExists(indexName).Do(ctx)
+	return es.client.IndexExists(indexName).AllowNoIndices(false).Do(ctx)
```

Comment:
For indexes with the "*" wild card, we need to add the "AllowNoIndices(false)". Otherwise, an empty set of indexes will be returned.